### PR TITLE
update solution template based on feedback

### PR DESCRIPTION
**Pull request by yungezz**
1. constraint ansible version >= 2.5
2. update readme
3. add constraint on domainNameLabel.
4. get custom script from deployment instead of github.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This repository contains scripts to deploy [Ansible](https://www.ansible.com/) o
 
 ## Customization
 - Ansible Version  
-  Pick up Ansible version to be installed. Available options: latest, 2.7.0, 2.6.5.
+  Fill in Ansible version later than 2.5.0 to be installed. You can get available release information from [here](https://github.com/ansible/ansible/releases), such as 2.7.4. Default is latest.
```
Comment (zikalino) [inline README.md]: - put (later than 2.5.0) in the brackets.
I would make **2.7.4** and **latest** bold

**Review by zikalino — APPROVED** [review]

**Comment by zikalino** [comment]
also I am wondering:
- should we allow to install ansible with **devel** branch? we could add version **devel**
- should we add option to install azure_preview_modules by default? **preview**?

File: README.md (hunk 1 above)
Comment (yungezz) [reply README.md]: resolved

**Comment by yungezz** [comment]
@zikalino solution template is providing official release, but not a test bed for devel branch.  for role, our role is not stable or mature enough to be installed by default. it still out of sync with upstream at lots of place.

**Pull request merged**